include/oneapi/dpl/pstl/hetero/dpcpp/sycl_traits.h

```diff
@@ -24,6 +24,8 @@
 #ifndef _ONEDPL_SYCL_TRAITS_H
 #define _ONEDPL_SYCL_TRAITS_H
 
+#include <oneapi/dpl/internal/binary_search_extension_defs.h>
```

Comment:
I believe it should be a relative patch, via ""

Reply:
I have removed this include.